core/src/main/java/org/opensearch/sql/expression/function/BuiltinFunctionName.java

```diff
@@ -44,6 +44,7 @@ public enum BuiltinFunctionName {
   SQRT(FunctionName.of("sqrt")),
   CBRT(FunctionName.of("cbrt")),
   TRUNCATE(FunctionName.of("truncate")),
+  MULTIPLYFUNCTION(FunctionName.of("multiply")),
```

Comment:
Please move this up to be below the `MOD` function so that it stays in alphabetical order.

Reply:
fixed